shared/src/exceptions/invalid-credentials-error/invalid-credentials-error.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  HttpCode,
+  CustomExceptionName,
+  CustomExceptionMessage,
+} from '~/common/enums/enums';
+
+class InvalidCredentialsError extends Error {
```

Comment:
in the shared folder, we need only shared thing
since we will use it only on the backend, move it there